styleguide/_themes/global/scss/modifiers.scss

```diff
@@ -167,7 +167,7 @@ hr {
 
   .half-padding-full,
   .standard-padding-full {
-    padding-left: $grid-gutter-width / 2;
-    padding-right: $grid-gutter-width / 2;
+    padding-left: $no-side-padding-left;
+    padding-right: $no-side-padding-right;
```

Comment:
This change has the potential of affecting a lot more than zoolander. This could impact www, and should be handled in isolation with a full QE process behind it.